src/types.ts

```diff
@@ -88,7 +88,7 @@ export interface TableLayout extends Omit<EngineAPI.IGenericHyperCubeLayout, 'qH
 
 export interface Cell {
   qText?: string;
-  qAttrExps: EngineAPI.INxAttributeExpressionValues;
+  qAttrExps?: EngineAPI.INxAttributeExpressionValues;
```

Comment:
I made this change as it reflects reality and is inline with the API spec [here](https://qlik.dev/apis/json-rpc/qix/schemas#%23%2Fdefinitions%2Fschemas%2Fentries%2FNxCell).

But it did end up not being necessery for the feature itself.

Reply:
that's fine